components/Comments/components/Form/Form.tsx

```diff
@@ -1,22 +1,27 @@
 import { Button, Error, Textarea } from 'components';
 import { useRouter } from 'next/router';
-import { FC } from 'react';
+import { FC, useEffect } from 'react';
 import { useForm } from 'react-hook-form';
 import { useAppDispatch } from 'store';
 import { useCreateCommentMutation } from 'store/apis';
 import { closeCommentsSidebar } from 'store/states';
 import { TSubmitFormEvent } from 'types';
+import { addKeyboardListener } from 'utils';
 
 import classes from './Form.module.scss';
 import { IFormProps } from './Form.types';
 
 export const Form: FC<IFormProps> = () => {
+  const isEnterPressed = addKeyboardListener('Enter');
```

Comment:
add ability to listen multiple avents addKeyboardListener(['Command+Enter', 'Control+Enter'], callback)